developing/learning/release_it.md

```diff
@@ -0,0 +1,8 @@
+# [Learning Resources](./README.md) / "Release It!"
+
+_Release It!_ was originally published in 2007 and was one of the most influential books at the beginning of the DevOps movement. The second edition, published in 2018, refreshes the content and practices for use in modern environments where DevOps and microservices are accepted norms.
+
```

Comment:
I found about this book recently when learning about the circuit breaker pattern, which I thought was very interesting. What do you think about adding a mention about that, maybe something like:

In this book, Michael Nygard popularized the [Circuit Breaker pattern](https://www.martinfowler.com/bliki/CircuitBreaker.html), which can be useful to prevent cascading failures when dealing with remote calls that fail or are unresponsive.

Reply:
Yupperz